grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -29,7 +29,9 @@ def section_actor_info(panels):
         panels.table_info("Table Id Info",
                           "Mapping from table id to actor id and table name",
                           [panels.table_target(f"{metric('table_info')}")], excluded_cols),
-
+        panels.table_info("Materialized View  Info",
```

Comment:
How about tracking the mv id as an additional column in `Table Id Info`?

Reply:
It's feasible, see
<img width="1450" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/58715567/8a870a3c-eaa6-45ba-bcfb-78b2f66826c1">.

But I prefer add another panel named `mv_info` to show mv_id, table_id and table_name, **without actor's information interference**.